k8s/demo/common/idam/idam.yaml

```diff
@@ -34,6 +34,23 @@ spec:
         aadIdentityName: idam
         environment:
           STRATEGIC_SERVICE_URL: https://idam-api.demo.platform.hmcts.net
+          TESTING_SUPPORT_ENABLED: true
```

Comment:
```suggestion
```

Reply:
testing support is required in demo (from Shrav)